creating_images/dot_net_core.adoc

```diff
@@ -0,0 +1,67 @@
+[[creating-images-using-dot-net-core]]
+= Using .NET Core with {product-title}
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[benefits-of-using-dot-net-core]]
+== Benefits of Using .NET Core
+
+$$.NET Core is a general purpose development platform featuring automatic memory
+management and modern programming languages. It allows users to
+build high quality applications efficiently.
+
+$$.NET Core is available on Red Hat Enterprise Linux (RHEL 7) and {product-title}
+via certified containers. .NET Core offers:
+
+* The ability to follow a microservices-based approach, where some components are
+built with .NET and others with Java, but all can run on a common, supported
+platform in Red Hat Enterprise Linux and {product-title}.
+* The capacity to more easily develop new .NET Core workloads on Windows;
+customers are able to deploy and run on either Red Hat Enterprise Linux or
+Windows Server.
+* A heterogeneous data center, where the underlying infrastructure is capable of
+running .NET applications without having to rely solely on Windows Server.
+* Access to many of the popular development frameworks such as .NET, Java, Ruby,
+and Python from within {product-title}.
+
+[[dot-net-core-supported-versions]]
+== Supported Versions
+
+* .NET Core framework version 1.0.
+* .NET is supported by Red Hat Enterprise Linux (RHEL) 7.
+* {product-title} (formerly called Red Hat OpenShift Enterprise) versions 3.2 and
+later are supported.
+
+[[dot-net-core-installing-images]]
+== Installing Images
+
+.About This Task
+This section explains how to install .NET Core 1.0 on Red Hat Enterprise Linux 7
+and perform a simple Hello World! script.
```

Comment:
they shouldn't need to install it, the default imagestreams should include the .NET imagestream so it should be present like any other imagestream (eg ruby, python, etc)